test/parallel/test-tls-interleave.js

```diff
@@ -31,7 +31,7 @@ const tls = require('tls');
 
 const fs = require('fs');
 
-const dir = common.fixturesDir;
+const dir = common.fixtures;
 const options = { key: fs.readFileSync(`${dir}/test_key.pem`),
```

Comment:
This object could be using methods `fixtures.readKey` instead of `fs.readFileSync` to get the keys and you could drop the const dir.